docs/pages/enterprise/sso.mdx

```diff
@@ -311,13 +318,14 @@ SAML and OIDC types:
 
 ## SSO Customization
 
-| Provider | YAML | Example |
-| - | - | - |
-| GitHub | `display: GitHub` | ![github](../../img/teleport-sso/github@2x.png) |
+| Provider  | YAML                 | Example                                               |
+|-----------|----------------------|-------------------------------------------------------|
+| Github    | `display: Github`    | ![github](../../img/teleport-sso/github@2x.png)       |
```

Comment:
"Github" should be "GitHub", since that's how the product is capitalized. Is `Github` enforced by `teleport`?

Reply:
No enforcement, just a typo.